dataeng/resources/remote-config.sh

```diff
@@ -91,11 +91,18 @@ unassume_role
 #         usernamePassword('ANALYTICS_VAULT_ROLE_ID', 'ANALYTICS_VAULT_SECRET_ID', 'analytics-vault');
 #     }
 # }
+# set path for config file which store token in each job workspace
+export VAULT_CONFIG_PATH= ${WORKSPACE}/vault-config/vault_config
+
+# write credentials to vault server to get the token
 vault write -field=token auth/approle/login \
     role_id=${ANALYTICS_VAULT_ROLE_ID} \
     secret_id=${ANALYTICS_VAULT_SECRET_ID} \
 | vault login -no-print token=-
 
+# set vault token 
```

Comment:
Maybe add a line here explaining why we have to do this bit, because it isn't in the vault documentation. It's a hack to get around vault not working the way we expected.